bower.json

```diff
@@ -4,9 +4,10 @@
   "authors": [
     "Konstantin Zudov <co@zudov.me"
   ],
+  "keywords": ["pursuit-deprecated"],
```

Comment:
The `pursuit-deprecated` keyword signals to Pursuit that the package should be marked as deprecated.